README.md

```diff
@@ -13,98 +13,32 @@ displays the contacts themselves, fetching them from content store:
 
-### Dependencies
+**Use GOV.UK Docker to run any commands that follow.**
 
-* Ruby 2.3.0
-* MySQL
-* [whitehall](https://github.com/alphagov/whitehall)
-  1. to access the organisations API and maintain parity with the organisations managed in whitehall.
-  2. to access the world locations API to provide the correct list of countries for the country dropdown in the address form
-* [signon](https://github.com/alphagov/signonotron2) - for managing user authentication
-* [content-store](https://github.com/alphagov/content-store) - for doing some
-  command-line based tasks to withdraw and redirect a contact
+### Before running the app
 
-### Running with govuk-docker
-
-Setup:
-
-```sh
-govuk-docker-run env RAILS_ENV=test env bin/rake db:create
-```
-
-Then run tests:
-
-```sh
-govuk-docker-run env RUNNING_IN_CI='true' bundle exec rspec
-```
-
-#### Previewing the application
-
-Seed the database, with ENV overrides so that `ImportOrganisations` doesn't attempt to fetch organisations from dev:
-
-```sh
-govuk-docker-run env GOVUK_APP_DOMAIN='https://www.gov.uk' env GOVUK_WEBSITE_ROOT='https://www.gov.uk' bin/rake db:seed
-```
-
-Run the app:
+The best way to get a database with good seed data is to replicate from Integration. Alternatively, you can use a combination of import commands and old seed data to seed the database, as follows:
 
 ```sh
-govuk-docker-up
-```
-
-Visit http://contacts-admin.dev.gov.uk/.
-
-You may see sometimes see a `GdsApi::TimedOutException` when performing a CRUD operation such as creating a contact.
-This usually means Publishing API is still starting up. Wait a few moments, then try again.
-
-### Running the application
-
-`./startup.sh`
-
-This runs `bundle install` to install dependencies and runs the app on port `3051`. When using the GOV.UK development VM it will be available at http://contacts-admin.dev.gov.uk/admin.
-
-By default, this application uses the GOV.UK preview environment for assets. To run against a local version of static, you need to set `STATIC_DEV` to
-"http://static.dev.gov.uk".
-
-### Running the test suite
-
-`bundle exec rake`
-
-The tests in this project rely upon [govuk-content-schemas](http://github.com/alphagov/govuk-content-schemas). By default these should be in the parent directory, otherwise you can specify their location with the `GOVUK_CONTENT_SCHEMAS_PATH` environment variable.
-
-### Tasks
-
-Rake tasks can be found in the standard location: `lib/tasks`. The implementation of many of these can be found in `app/tasks` and `app/interactors` so they can also be run from the console.
-
-### Database setup
-
-The best way to get a database with good seed data is to use a dump from preview; alternatively you can load the database schema and use the old initial seed data:
-
-```
-bundle exec rake db:schema:load
-bundle exec rake db:seed
+env GOVUK_APP_DOMAIN='https://www.gov.uk' env GOVUK_WEBSITE_ROOT='https://www.gov.uk' bin/rake db:seed
```

Comment:
```suggestion
env GOVUK_APP_DOMAIN='https://www.gov.uk' GOVUK_WEBSITE_ROOT='https://www.gov.uk' bin/rake db:seed
```

`env` accepts multiple environment variables, so it shouldn't be necessary to run `env` again.

Reply:
Indeed, nice catch. I've snuck this in.